src/services/user.service.js

```diff
@@ -0,0 +1,31 @@
+const { User } = require('./../models/User.model');
+
+const getAllUsers = async () => {
+  return User.findAll();
```

Comment:
Consider adding `await` before `User.findAll()` to ensure that the asynchronous operation is completed before returning the result. This will make the code more consistent with the use of `await` in other service functions.